cardano-cli/test/Test/golden/shelley/node-pool/kes.skey

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
What are all of these new golden files for? Are they related to this PR?

Reply:
Oops.  They weren't supposed to be included.  Will remove.